fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/orderby/OrderByColumn.java

```diff
@@ -51,6 +51,17 @@ public OrderByColumn(PostAggregation postAggregation, SortDirection direction) {
     }
 
     /**
+     * Constructor.
+     *
+     * @param dateTime  a dateTime
```

Comment:
Since this constructor is quite different from the rest, and it takes very generic parameters, it should get more description as to why it exists and what it's intended to be used for. Additionally, the comment for this parameter is not great... if I give it, for example, `2017-02-01T10:25`, something tells me it wouldn't do the right thing.

If we could have a way to create this that _didn't_ require the user to specify the column name, that would be great. A static factory method that takes a direction, for example.